apps/guide/src/content/04-popular-topics/04-webhooks.mdx

```diff
@@ -0,0 +1,227 @@
+#### Using the WebhookClient constructor
+
+If you are not using a bot client, you can get a webhook by creating a new instance of _`WebhookClient`_ and passing the _`id`_ and _`token`_ into the constructor. These credentials do not require you to have a bot application, but it also offers limited information instead of fetching it using an authorized client.
+
+<CH.Code>
+
+```js
+const webhookClient = new WebhookClient({ id: 'id', token: 'token' });
+```
+
+</CH.Code>
+
+You can also pass in just a _`url`_:
+
+<CH.Code>
+
+```js
+const webhookClient = new WebhookClient({ url: 'https://discord.com/api/webhooks/id/token' });
+```
+
+</CH.Code>
+
+## Creating webhooks
+
+### Creating webhooks through server settings
+
+You can create webhooks directly through the Discord client. Go to Server Settings, and you will see an _`Integrations`_ tab.
+
+![Integrations tab](/assets/integrations-tab.png)
+
+If you already have created a webhook, the webhooks tab will look like this; you will need to click the _`View Webhooks`_ button.
+
+![Integrations tab](/assets/integrations-view-tab.png)
+
+Once you are there, click on the _`Create Webhook`_ / _`New Webhook`_ button; this will create a webhook. From here, you can edit the channel, the name, and the avatar. Copy the link, the first part is the id, and the second is the token.
+
+![Creating a Webhook](/assets/webhook.png)
+
+### Creating webhooks with discord.js
+
+Webhooks can be created with the <DocsLink type="class" parent="TextChannel" symbol="createWebhook" brackets /> method.
+
+<CH.Code>
+
+```js
+channel
+	.createWebhook({ name: 'Username', avatar: 'https://guide.discordjs.dev/assets/discordjs.png' })
+	.then((webhook) => console.log(`Created webhook ${webhook}`))
+	.catch(console.error);
+```
+
+</CH.Code>
+
+## Editing webhooks
+
+You can edit Webhooks and WebhookClients to change their name, avatar, and channel using <DocsLink type="class" parent="Webhook" symbol="edit" brackets />.
+
+<CH.Code>
+
+```js
+webhook
+	.edit({ name: 'Username', avatar: 'https://guide.discordjs.dev/assets/discordjs.png', channel: '123456789012345678' })
+	.then((webhook) => console.log(`Edited webhook ${webhook}`))
+	.catch(console.error);
+```
+
+</CH.Code>
+
+## Using webhooks
+
+Webhooks can send messages to text channels, as well as fetch, edit, and delete their own. These methods are the same for both _`Webhook`_ and _`WebhookClient`_.
+
+### Sending messages
+
+Webhooks, like bots, can send up to 10 embeds per message. They can also send attachments and normal content. The <DocsLink type="class" parent="Webhook" symbol="send" brackets /> method is very similar to the method used for sending a message to a text channel. Webhooks can also choose how the username and avatar will appear when they send the message.
+
+Example using a _`WebhookClient`_:
+
+<CH.Code>
+
+```js
+import { EmbedBuilder, WebhookClient } from 'discord.js';
+import { webhookId, webhookToken }  from './config.json' assert { type: "json" }
```

Comment:
Destructuring a JSON import doesn't seem to work in ESM, unless there's something I'm missing.

```
import { webhookId, webhookToken }  from './config.json' assert { type: "json" }
         ^^^^^^^^^
SyntaxError: The requested module './config.json' does not provide an export named 'webhookId'
```

Reply:
You're absolutely right. Fixed with 9f62034d16ccc8970aad7ebca5fa6ec3e16af56b.